pkg/database/migrations.go

```diff
@@ -1842,6 +1842,38 @@ func (db *Database) Migrations(ctx context.Context) []*gormigrate.Migration {
 				return tx.Exec(`ALTER TABLE realms DROP COLUMN IF EXISTS alternate_sms_templates`).Error
 			},
 		},
+		{
+			ID: "00078-AddUserDefaultSMSTemplate",
+			Migrate: func(tx *gorm.DB) error {
+				sqls := []string{
+					`ALTER TABLE users ADD COLUMN IF NOT EXISTS remember_last_used_sms_template BOOL`,
+					`UPDATE users SET remember_last_used_sms_template = TRUE WHERE remember_last_used_sms_template IS NULL`,
+					`ALTER TABLE users ALTER COLUMN remember_last_used_sms_template SET DEFAULT TRUE`,
+					`ALTER TABLE users ALTER COLUMN remember_last_used_sms_template SET NOT NULL`,
+					`ALTER TABLE users ADD COLUMN IF NOT EXISTS default_sms_template_label VARCHAR(255)`,
```

Comment:
We need to add some validation or a database constraint here. Either:

- Changing an SMS label should fail if a user is depending on it
- Changing an SMS label clears the default for any users